map.go

```diff
@@ -432,6 +432,17 @@ func (spec *MapSpec) createMap(inner *sys.FD, opts MapOptions) (_ *Map, err erro
 
 	fd, err := sys.MapCreate(&attr)
 
+	// BPF_MAP_TYPE_RINGBUF's max_entries must be a power-of-2 multiple of kernel's page size.
```

Comment:
Thanks for your contribution!

Can we move this error check to `handleMapCreateError()`? `handleMapCreateError()` is a central place to extend the returned error with more information.

Reply:
done